### PR TITLE
PL 🇵🇱 translation

### PR DESCRIPTION
**Pull request by Cs4K1Sr4C**

### CODE REVIEW

**Review by Cs4K1Sr4C — COMMENTED** [review]
ATi18n completely removed from the locales folder. :+1:

**Comment by miurla** [comment]
@Cs4K1Sr4C 
Thanks for the quick response! Too cool, man!
Deploying.

**Comment by Cs4K1Sr4C** [comment]
@miurla 
Always you're welcome! I hope today I can implement RTL to the project. I'll let you know about the process.

**Comment by miurla** [comment]
Have you already started on RTL? That's amazing!
Thanks for everything.

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ This is a port of [babyagi](https://github.com/yoheinakajima/babyagi) with [Lang
 - [x] Exporting Execution Results
 - [x] Execution history
 - [x] Faster speeds and fewer errors. ([😺 BabyCatAGI](https://twitter.com/yoheinakajima/status/1657448504112091136))
-- [x] i18n support ( '🇧🇷', '🇩🇪', '🇺🇸', '🇪🇸', '🇫🇷', '🇮🇳', '🇭🇺', '🇯🇵', '🇷🇺', '🇹🇭',)
+- [x] i18n support ( 🇧🇷, 🇩🇪, 🇺🇸, 🇪🇸, 🇫🇷, 🇮🇳, 🇭🇺, 🇯🇵, 🇷🇺, 🇹🇭, ... and much more)
```
Comment (miurla) [inline README.md:23]: Thx 🙏

**Pull request merged**